tests/countries/test_antigua_and_barbuda.py

```diff
@@ -0,0 +1,227 @@
+            name,
+            "2021-04-05",
+            "2022-04-18",
+            "2023-04-10",
+            "2024-04-01",
+            "2025-04-21",
+        )
+        self.assertHolidayName(name, range(1955, 2050))
+
+    def test_labour_day(self):
+        name = "Labour Day"
+        self.assertHolidayName(
+            name,
+            "2021-05-03",
+            "2022-05-02",
+            "2023-05-01",
+            "2024-05-06",
+            "2025-05-05",
+        )
+        self.assertHolidayName(name, range(1955, 2050))
+
+    def test_whit_monday(self):
+        name = "Whit Monday"
+        self.assertHolidayName(
+            name,
+            "2021-05-24",
+            "2022-06-06",
+            "2023-05-29",
+            "2024-05-20",
+            "2025-06-09",
+        )
+        self.assertHolidayName(name, range(1955, 2050))
+
+    def test_caribbean_community_day(self):
+        name = "Caribbean Community (Caricom) Day"
+        self.assertHolidayName(
+            name,
+            "1995-07-03",
+            "2000-07-03",
+            "2004-07-05",
+            "2005-07-04",
+        )
+        self.assertNoHoliday(
+            "2010-07-05",
+            "2015-07-06",
+            "2020-07-06",
+        )
+        self.assertHolidayName(name, range(1955, 2006))
+        self.assertNoHolidayName(name, range(2006, 2050))
+
+    def test_carnival_monday(self):
+        name = "Carnival Monday"
+        self.assertHolidayName(
+            name,
+            "2021-08-02",
+            "2022-08-01",
+            "2023-08-07",
+            "2024-08-05",
+            "2025-08-04",
+        )
+        self.assertHolidayName(name, range(1955, 2050))
+
+    def test_carnival_tuesday(self):
+        name = "Carnival Tuesday"
+        self.assertHolidayName(
+            name,
+            "2021-08-03",
+            "2022-08-02",
+            "2023-08-08",
+            "2024-08-06",
+            "2025-08-05",
+        )
+        self.assertHolidayName(name, range(2006, 2050))
+        self.assertNoHolidayName(name, range(1955, 2006))
+
+    def test_independence_day(self):
+        name = "Independence Day"
+        self.assertHolidayName(name, (f"{year}-11-01" for year in range(1955, 2050)))
+        dt = (
+            "2008-11-03",
+            "2009-11-02",
+            "2014-11-03",
+            "2015-11-02",
+            "2020-11-02",
+            "2025-11-03",
+        )
+        self.assertHolidayName(f"{name} (observed)", dt)
+        self.assertNoNonObservedHoliday(dt)
+
+    def test_national_heroes_day(self):
+        name = "National Heroes Day"
+        self.assertHolidayName(
+            name,
+            (
+                "2005-12-09",
+                "2006-12-11",
+                "2007-12-09",
+                "2008-12-09",
+                "2009-12-09",
+                "2010-12-09",
+                "2011-12-09",
+                "2012-12-10",
+                "2013-12-09",
+            ),
+        )
```

Comment:
```suggestion
        self.assertHolidayName(
            name,
            "2005-12-09",
            "2006-12-11",
            "2007-12-09",
            "2008-12-09",
            "2009-12-09",
            "2010-12-09",
            "2011-12-09",
            "2012-12-10",
            "2013-12-09",
        )
```

Reply:
Superceded by KJ's comment below which combines this with `test_sir_vere_cornwall_bird_snr_day`